cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -535,4 +535,16 @@ protected void setUsernameAndPassword(String username, PasswordSecretSource pass
     public PodDisruptionBudget generatePodDisruptionBudget() {
         return createPodDisruptionBudget();
     }
+
+    @Override
+    protected String getServiceAccountName() {
+        return initContainerServiceAccountName(cluster);
+    }
+
+    /**
+     * Get the name of the connect service account given the name of the {@code connectResourceName}.
+     */
+    public static String initContainerServiceAccountName(String connectResourceName) {
```

Comment:
This probably should be renamed. It is called `initContainer...` because in Kafka it was used mainly because of the init container. But there are no init containers here, Zookeeper or in Mirror Maker.